Makefile

```diff
@@ -51,17 +51,17 @@ test-clean:
 	rm -rf .pytest_cache/
 
 test:
-	tox -e "pyupgrade,black,mypy,flake8,docstrings"
-
-test-full:
 	tox
 
 test-short:
-	tox -e "pyupgrade,black,mypy,flake8,docstrings,py36"
+	tox -e "pyupgrade,isort-format,format,mypy,flake8,docstrings,py38"
```

Comment:
this should be isort-check and black right..... this is test, not change.

i also think of lint as just telling me something is wrong, not actually changing it.

I am all for renaming a bunch of these so it makes sense.. lets have a thread in slack and decide what makes sense